### DISCUSSION TITLE
Highlight when a replacement value is an empty string similar to when it's null

### DISCUSSION THREAD

**Discussion by xt0rted** [datalust/seq-tickets#2247] [Ideas]
When a replacement value is `null` it's displayed and styled different from the message which makes it clear there was no value. It'd be really helpful if something similar could be done for empty strings. There's times where I miss that a value was provided but empty (or only whitespace) just by looking at the message. Adding quotes around string values that could be empty helps a bit, but that's an extra step you have to always remember to do.

![image](https://github.com/user-attachments/assets/00309544-97ca-44e3-8822-5ea108cf6ec0)

For now I'm using Stylus with the following css to highlight missing values in the message, but it doesn't help if it's only whitespace or style the value when you expand the entry.

```css
.token-string:empty::after {
  content: "empty";
  color: #94d4f3;
}
```

![image](https://github.com/user-attachments/assets/129f2194-13b4-4c0e-8d6c-84893ed74f52)

**Comment by nblumhardt** [comment]
Thanks for the suggestion :+1:
